### PR TITLE
Refactor: Migrate to use the `rest` package in util/

### PR DESCRIPTION
**Pull request by breeswish**
This is one step to drop the usage of `pkg/apiserver/utils`.

### CODE REVIEW

File: ui/lib/client/index.tsx [hunk 1]
```diff
@@ -62,20 +60,23 @@ function applyErrorHandlerInterceptor(instance: AxiosInstance) {
     let errCode: string
     let content: string
     if (err.message === 'Network Error') {
-      errCode = 'error.network'
+      errCode = 'common.network'
```
Comment (baurine) [inline ui/lib/client/index.tsx:63]: Change the i18n key needs to change the translation as well.

File: ui/lib/client/index.tsx [hunk 2]
```diff
@@ -62,20 +60,23 @@ function applyErrorHandlerInterceptor(instance: AxiosInstance) {
...
-        response?.data?.message || err.message || i18next.t(ERR_CODE_OTHER)
+      content = String(
+        response?.data?.message || err.message || 'Internal error'
+      )
     }
     err.message = content
     err.errCode = errCode
 
-    if (errCode === 'error.api.unauthorized') {
+    if (errCode === 'common.unauthenticated') {
```
Comment (baurine) [inline ui/lib/client/index.tsx:79]: Change the i18n key needs to change the translation as well.

**Review by baurine — APPROVED** [review]
lgtm, just needs to add the missed translation

**Comment by breeswish** [comment]
There is a merging mistake that missed something. I need some manual checks again to see whether other things are missed.

**Comment by breeswish** [comment]
Manual tests are finished.

**Pull request merged**